content/blog/check_my_repo.md

```diff
@@ -0,0 +1,150 @@
+
+#### Security
+
+A SECURITY.md is where a security policy goes, a place to instruct users how to report a security vulnerability. This allows users to easily report and maintainers to quickly take actions upon vulnerabilities.
+
+#### Support
+
+A SUPPORT.md file is used to describe how the project is being maintained and how responsible a maintainer will be on issues. It can include: how and where a user can get help and support, which communication channels should be used and what not to do to avoid communication noise. This file helps both users and maintainers to communicate to each other having their needs to get into the right channel.
+
+Now imagine a project with all that. Will it help to build and keep a community? Will noise in communication be mitigated? Will it save tons of maintainers time to help people onboard and solve issues?
+
+Have I convinced you?
+
+So get a check-my-repo to call &quot;my own&quot; and make your friends envy you for how beautiful and healthy your repositories are.
+
+## Conclusion
+
+Creating and maintaining an open source project is very rewarding. Creating collaboratively is an incredible experience and has the intrinsic potential to take such creation into possibilities that one person alone or a small group could never achieve.
+
+But working openly and collaboratively is also a challenge for the maintainers and a responsibility for the community to ensure that this space is equitable.
+
+Tools like check-my-repo help in a practical way with the maintenance of repositories by automatically checking whether projects comply with minimum best practices.
+
+But it doesn&#39;t have to stop there.
+
+There is a lot of work ahead as shown by the results of surveys on the health of open source communities, which end up reflecting the worst of the technology industry, since open source is usually practiced voluntarily in the spare time of its community.
+
+That&#39;s why ensuring that these standards are used is so important for the continuation and improvement of a practice that has so much potential.
+
+
+
+_\*This text is based on a talk for_ [_Pizza Python Hamburg 2021_](https://hamburg.python.pizza/) _https://www.youtube.com/watch?v=hXKdtY8gHbs&amp;t=9240s_
+
+
+
+[1]: In February 2017 Wired magazine published the following article based on a GitHub survey, analizing diversity in open source https://www.wired.com/2017/06/diversity-open-source-even-worse-tech-overall/ . Although outdated, the article still reflects several issues still happening today. Recently, Linux Foundation together with several associations, including CHAOSS project, conducted another survey https://www.linuxfoundation.org/blog/addressing-diversity-equity-and-inclusion-in-2021-and-beyond/ , which shows several improvements related to diversity, equity and social justice awareness. Unfortunately the survey does not explicitly show in numbers if the 3% was improved or not, so, for now, we will still use it as a measurement.
```

Comment:
analyzing.

also use markdown for links and consider using archive.org.  should probably link to CHAOSS, LF, etc.

rather strange that the linter didn't automatically fix the issues.  I swear I set something up.  how did you commit this changeset?

Reply:
I am lost in this o_O is [1]: not a markdown for footnote? 
What do you mean by that? "consider using archive.org. should probably link to CHAOSS, LF, etc." it is a LF research. Lost lol.
I committed via git...

Reply:
- I mean instead of bare links (`https://...`) use markdown `[text](https://...)`
- may want to change the wired article link and LF foundation link to point to archive.org copy. up to you though
- link to CHAOSS and LF means exactly that. `[CHAOSS](http://chaoss.url)`